File: Med-App-API/Models/Appointments.cs
using System;

namespace Med_App_API.Models
{
    public class Appointments
    {
        public int Id { get; set; }
        public int PatientId { get; set; }
        public Patient Patient { get; set; }
        public int PhysicianId { get; set; }
        public Physician Physician { get; set; }
        public DateTime TimeofAppointment { get; set; }
        public string TypeOfAppointment { get; set; }
    }
}

File: Med-App-SPA/src/app/_models/appointments.ts
import {Time} from "@angular/common";

export interface Appointments {
  typeOfAppointment: string;
  id: number
  patientId: number
  patientFullName:string
  physicianId: number
  physicianFullName:string
  startOfAppointment: Time
  endOfAppointment: Time
  description?: string
}
